src/main/java/com/getbase/models/User.java

```diff
@@ -25,6 +25,7 @@
   protected @JsonView(Views.ReadOnly.class) Group group;
   protected @JsonView(Views.ReadOnly.class) List<Role> roles = new ArrayList<Role>();
   protected @JsonView(Views.ReadOnly.class) Boolean invited;
+  protected @JsonView(Views.ReadOnly.class) Long zendeskUserId;
```

Comment:
the property itself should also be marked/warned as subject to change

Reply:
done